### PR TITLE
rpi-kernel,rpi-firmware: update to 5.15.61. [ci skip]

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **briefly**

I've built rpi4-kernel and rpi-firmware, and created an image for my CM4. It boots, haven't tested anything more so far. I sadly don't have any other Pi products to test on right now, so I'd need some other people to help on this.

cc @Piraty as current maintainer

Resolves https://github.com/void-linux/void-mklive/issues/264

### CODE REVIEW

File: srcpkgs/rpi-kernel/template [hunk 1]
```diff
@@ -12,22 +12,22 @@
...
-hostmakedepends="perl kmod uboot-mkimage openssl-devel bc bison flex"
+hostmakedepends="perl kmod uboot-mkimage openssl-devel bc bison flex xz"
 makedepends="ncurses-devel"
 maintainer="Piraty <piraty1@inbox.ru>"
 homepage="http://www.kernel.org"
 license="GPL-2.0-only"
 short_desc="Linux kernel for Raspberry Pi zero/1 (${version%.*} series [git ${_gitshort}])"
 distfiles="https://github.com/raspberrypi/linux/archive/${_githash}.tar.gz"
-checksum=f3ee7782c90ecb3e468e89602248a1de45b466d5d4c1af4748f7ca088c0fcab3
+checksum=64ad74084fa44abe8689564071df5729ded4c589
```
Comment (Piraty) [inline srcpkgs/rpi-kernel/template:30]: checksum doesn't matcht

File: srcpkgs/rpi-kernel/template [hunk 2]
```diff
@@ -206,10 +206,6 @@ do_install() {
 	mkdir -p ${hdrdest}/net/mac80211/
 	cp net/mac80211/*.h ${hdrdest}/net/mac80211
 
-	# add dvb headers for external modules
```
Comment (Piraty) [inline srcpkgs/rpi-kernel/template:209]: align with `srcpkgs/linux5.15/template`. align with mainline template on other occasions is wise too.

**Review by Piraty — CHANGES_REQUESTED** [review]
thanks for working on this!

**Comment by paper42** [comment]
Tested on Raspberry Pi 4 4GB, it's been running for 3 days without any issues.

File: srcpkgs/rpi-kernel/template (hunk 1 above)
Comment (jcgruenhage) [reply srcpkgs/rpi-kernel/template:30]: I seem to have mis-pasted something here, sorry

File: srcpkgs/rpi-kernel/template (hunk 2 above)
Comment (jcgruenhage) [reply srcpkgs/rpi-kernel/template:209]: wdym?

**Comment by tsjk** [comment]
This is awesome. Looking forward to the merge.

File: srcpkgs/rpi-kernel/template [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 #
 # We track the latest Raspberry Pi LTS kernel as that is what is used in the
-# official Raspbian distribution. This is currently 5.10:
+# official Raspberry Pi OS distribution. This is currently 5.15:
 #
-# https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=288234
+# https://forums.raspberrypi.com/viewtopic.php?t=322879
 #
 # Commit hash is picked from latest tag [1], if appropriate, or from latest
 # "Merge remote-tracking branch 'stable/linux-5.10.y' into rpi-5.10.y" commit.
```
Comment (Piraty) [inline srcpkgs/rpi-kernel/template:9]: `5.10` -->`5.15` ...

File: srcpkgs/rpi-kernel/template (hunk 2 above)
Comment (Piraty) [reply srcpkgs/rpi-kernel/template:209]: work of many board kernels is duplicated work of the mainline template, so taking inspiration from the corresponding mainline template is nice , avoids hard-to-read diffs later on too

**Pull request merged**